src/main/index.js

```diff
@@ -7,6 +7,16 @@ import '../printer';
 
 export { downloadBlob } from './utils';
 
+/**
+ * @typedef {Object} TileGrid
+ * @property {!Array<number>} resolutions Resolutions. The array index of each
```

Comment:
I don't get the use of `!` here. If it is a user input, shouldn't we assume it could sometimes be undefined/null ?

Reply:
resolutions are mandatory otherwise it won't work.